source/OutfitInfoDisplay.cpp

```diff
@@ -352,7 +352,7 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	if(outfit.Ammo())
 	{
 		attributeLabels.emplace_back("ammo:");
-		attributeValues.emplace_back(outfit.Ammo()->Name());
+		attributeValues.emplace_back(outfit.Ammo()->DisplayName());
```

Comment:
I disagree with this change, as it means the player can think they have the ammo ("yes, I have the outfit 'florpwax missile' installed") but the game may disagree ("ship has outfit 'florpwax missile prototype A' installed"). The player has no way to reconcile the difference; every reference they can see in-game to the outfit/ammo says "florpwax missile."

Reply:
I agree that we should give warnings if Ammo is being given a different DisplayName(); that just shouldn't happen. We should even remove the DisplayName for Outfits that can be used as ammo.